be/src/io/cache/fs_file_cache_storage.h

```diff
@@ -102,6 +102,12 @@ class FSFileCacheStorage : public FileCacheStorage {
 
     void load_cache_info_into_memory(BlockFileCache* _mgr) const;
 
+    void check_consistency(
+            BlockFileCache* _mgr,
```

Comment:
Better not couple this interface with FileCacheManager. The method of checking here is that the storage can return a list with comprehensive "file" information (including path, size, creation time, etc.), preferably in an ordered manner. Let the FileCacheManager, which depends on this storage, perform its own forward and reverse checks to see if there is any extra or missing data in the storage.

Reply:
Thank you for the correction, I will make the modifications promptly.